.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+  - node
+  - lts/*
```

Comment:
Can we add the test command here?

Reply:
@akonchady We could, but it should default to `npm test` per https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#default-build-script which we've defined here:
https://github.com/akonchady/disable-eslint-issues-cli/blob/ba487e36d5c12420753e806bb64e76e3f3088164/package.json#L10

Reply:
Good to know 👍